<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtConflictDialog</class>
 <widget class="QDialog" name="QtConflictDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>211</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Conflict</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtClient.qrc">
    <normaloff>:/Resources/icon_cf.png</normaloff>:/Resources/icon_cf.png</iconset>
  </property>
  <widget class="QCommandLinkButton" name="downloadButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Server Version</string>
   </property>
   <property name="icon">
    <iconset resource="qtClient.qrc">
     <normaloff>:/Resources/icon_dl.png</normaloff>:/Resources/icon_dl.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="description">
    <string>Name:
Size:
Last Modification:</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="uploadButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Local Version</string>
   </property>
   <property name="icon">
    <iconset resource="qtClient.qrc">
     <normaloff>:/Resources/icon_ul.png</normaloff>:/Resources/icon_ul.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="description">
    <string>Name:
Size:
Last Modification:
Last synchronized Version:</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="keepButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>solve conflict manually</string>
   </property>
   <property name="description">
    <string>rename the local version to .mc_conflicted</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="skipButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>skip this time</string>
   </property>
   <property name="description">
    <string>check again on next run</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="recursiveBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>and below</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="dirBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>choose the same for all conflicts in the current directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;font color=&quot;gray&quot;&gt;(use at your own risk)&lt;/font&gt;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>uploadButton</tabstop>
  <tabstop>downloadButton</tabstop>
  <tabstop>keepButton</tabstop>
  <tabstop>skipButton</tabstop>
  <tabstop>dirBox</tabstop>
  <tabstop>recursiveBox</tabstop>
 </tabstops>
 <resources>
  <include location="qtClient.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dirBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>recursiveBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
